README.md

```diff
@@ -43,51 +48,49 @@ This is the recommended stable release, as it has been thoroughly tested.
 
 [Download the stable release here.](https://github.com/flybywiresim/a32nx/releases/latest/download/flybywiresim-a32nx.zip)
 
-You can see the changelog on the releases page: [View Here.](https://github.com/flybywiresim/a32nx/releases)
+You can view the changelog on the [releases page.](https://github.com/flybywiresim/a32nx/releases)
 
 #### Unstable Master Branch Build
 
 This has the latest features, but is much more unstable, use at your own risk.
 
-[Download developer build here.](https://github.com/flybywiresim/a32nx/releases/download/vmaster/A32NX-master.zip)
+[Download development build here.](https://github.com/flybywiresim/a32nx/releases/download/vmaster/A32NX-master.zip)
 
-[View info about the latest build here.](https://github.com/flybywiresim/a32nx/releases/tag/vmaster)
+[View information regarding the latest build here.](https://github.com/flybywiresim/a32nx/releases/tag/vmaster)
 
 #### Unstable Master Branch Build (with custom autopilot)
 
-This version is the same as the regular master/development version, but with the WIP custom fly-by-wire, autopilot and autothrust system. Expect issues with autopilot, engines, autothrust and the PFD if you intend to use this version. No support will be provided via Discord.
-
-[Download custom autopilot development build here.](https://api.flybywiresim.com/api/v1/download?url=https://flybywiresim-packages.b-cdn.net/vmaster-cfbw-cap/A32NX-master-cfbw-cap.zip)
+This version is the same as the regular master/development version. However, this version comes with the (work in progress) LNAV and flight plan manager. **Expect issues with navigation and flight planning.** No support will be provided via Discord.

```

Comment:
Let's just avoid having to update the readme everytime experimental is used for something different.

```suggestion
#### Experimental Branch Build

This branch is used from time-to-time for experimental features under heavy development. It's content will vary over time. No support will be provided via Discord, although users of the branch are advised to keep themselves updated with development progress via Discord.
```